sdk/src/main/java/io/dapr/client/DaprClientHttp.java

```diff
@@ -179,6 +186,14 @@ public Mono<Void> publishEvent(PublishEventRequest request) {
     }
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public <T> Mono<BulkPublishResponse<T>> publishEvents(BulkPublishRequest<T> request) {
+    return DaprException.wrapMono(new UnsupportedOperationException());
```

Comment:
I assume this will be implemented next, right?

Reply:
@artursouza as we discussed previously, we were saying that HTTP impl is not needed, and that HTTP Client is going to be deprecated. https://github.com/dapr/java-sdk/pull/790